src/backend/efiling-hub-api/src/main/resources/logback-splunk.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    ​
+    <!-- You can override this to have a custom pattern -->
+    <property name="CONSOLE_LOG_PATTERN"
+              value="%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}"/>
+
+    <appender name="http" class="com.splunk.logging.HttpEventCollectorLogbackAppender">
+        <url>${SPLUNK_URL}</url>
+        <token>${SPLUNK_TOKEN}</token>
+        <source>jag-efiling-hub-api</source>
+        <sourcetype>jag-efiling-access-logs</sourcetype>
```

Comment:
Should that be `jag-efiling-logs` I don't know how we are configured on the other one...

Reply:
jag-efiling-api in the other